csm/CommsMan.py

```diff
@@ -0,0 +1,95 @@
+import threading
+import queue
+import time
+
+class CommsMan():
+
+    def __init__(self, commsQueue, termCommsQueue):
+        self.sysQueueSend = commsQueue
+        self.sysQueueRecv = queue.Queue(maxsize=1)
+        self.btQueue = queue.Queue(maxsize=1)
+        self.terminateComms = queue.Queue(maxsize=1)
+        self.testSendOutput = []
+
+    def sendMessageToRemote(self, message):
+        """
+        External function which enables users to message CommsMan to send
+        message to the remote interface via Bluetooth for user operation.
+        :param message: Contents of message to send to remote (String)
+        """
+        self.sysQueueRecv.put(message)
+
+    def terminateCommsMan(self):
+        """
+        External function to terminate CommsMan thread operations.
+        """
+        try:
+            self.terminateComms.put(-1)
+            return 0
+        except:
+            return -1
```

Comment:
When would this happen? Could you document it here with comments?

Reply:
Good note, important to comment in, reminded me of an addition needed as well